src/components/CreditCardDetails/components/scheme-icons/card-scheme-map.js

```diff
@@ -0,0 +1,35 @@
+import Amex from './icons/amex.svg';
```

Comment:
In this file you are basically duplicating what we already have in [`card-scheme-icons.js`](https://github.com/sumup/circuit-ui/blob/master/src/components/CreditCardDetails/components/scheme-icons/card-scheme-icons.js), right? Since that module also adds accessibility attributes etc. and we don't want to duplicate stuff, I think it should be fine for you to just use the other module instead. Can you do that?